app/src/components/utils/Snackbar.jsx

```diff
@@ -1,32 +1,45 @@
 import React, { PropTypes, Component } from 'react';
-import { connect } from 'react-redux';
+import { connect, dispatch } from 'react-redux';
 import SnackbarUI from 'material-ui/Snackbar';
 import { emptySnackbar } from '../../actions/actionCreators';
 export class Snackbar extends Component {
 
     render() {
-        const message = this.props.message;
+        const { message, retryAction } = this.props.snackbar;
+        const retryProps = {
+            action: 'Retry',
+            onActionTouchTap: retryAction
+        }
```

Comment:
Missing a semicolon. ;) Not really critical but i've seen it in a few places now. We could consider adding a linter to make sure the style is consistent.